packages/estatico-boilerplate/src/assets/js/helpers/app.js

```diff
@@ -3,7 +3,7 @@
  *
  * @license APLv2
  */
-import $ from 'jquery';
+import EventDelegate from 'dom-delegate';
```

Comment:
When i look at the documentation, it seems that we should be using a named export `Delegate` instead of the default export when using `new`: https://www.npmjs.com/package/dom-delegate#usage